pkgs/by-name/em/emmy-lua-code-style/package.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lib,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (self: {
+  pname = "emmy-lua-code-style";
+  version = "1.5.4";
+
+  src = fetchFromGitHub {
+    owner = "CppCXY";
+    repo = "EmmyLuaCodeStyle";
+    rev = "refs/tags/${self.version}";
+    hash = "sha256-UqAAUZDkdQyl8qkp1Au97VlU4DKZwVokB+IGmyCp+Ek=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  passthru.updateScript = nix-update-script {};
+
+  meta = {
+    homepage = "https://github.com/CppCXY/EmmyLuaCodeStyle";
+    changelog = "https://github.com/CppCXY/EmmyLuaCodeStyle/releases/tag/${self.version}";
+    description = "fast, powerful, and feature-rich Lua formatting and checking tool.";
+    mainProgram = "CodeFormat";
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

question: Can this only be built on linux and darwin, or can we enable it for all unix platforms?

```suggestion
    platforms = lib.platforms.unix;

---

I would image it was only ever tested on Linux and Darwin, but might work elsewhere too.

---

In that case, let's set this to `unix`, and we can change it if we discover it's broken on other platforms.